README.md

```diff
@@ -21,6 +21,7 @@ app.addEventListener('error', (evt) => {
 const router = new Router<AppState>();
 
 // Apply sessions to your Oak application. You can also apply the middleware to specific routes instead of the whole app.
+// Without params, default MemoryStore is used. See below Storage chapter for more info.
```

Comment:
Thanks! Grammar nitpick, change to `See the Storage chapter below for more info`